.github/workflows/update-docs-on-docs-commit.yml

```
@@ -0,0 +1,42 @@
+# This workflow is centrally managed in https://github.com/asyncapi/.github/
+# Don't make changes to this file in this repo as they will be overwritten with changes made to the same file in above mentioned repo
+
+# The given workflow is responsible for generating docs and creating PR with them when there is a commit with docs: prefix
+
+# This workflow will be updated in all repos with the topic autogenerate-docs
+
+name: 'Update generated parts of documentation on docs: commits'
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docs-gen:
+    name: 'Generate docs and create PR'
+    runs-on: ubuntu-latest
+    # PR should be created within this GH action only if it is a docs: commit
+    # Otherwise it will conflict with release workflow
+    if: startsWith(github.event.commits[0].message, 'docs:')
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

```suggestion
      - name: Check package-lock version
        uses: asyncapi/.github/.github/actions/get-node-version-from-package-lock@master
        id: lockversion

      - name: Use Node.js
        uses: actions/setup-node@v3
        with:
          node-version: "${{ steps.lockversion.outputs.version }}"
          cache: 'npm'
          cache-dependency-path: '**/package-lock.json'
```

this is a solution that we use not, that assures we always setup a node version that is compatible with package-lock version